ci/jenkins/Jenkins_steps.groovy

```diff
@@ -1168,6 +1192,28 @@ def docs_publish_beta() {
     }]
 }
 
+// This is for uploading website artifacts to S3 bucket
+// Assumes you have run docs_full_website function
+def docs_upload_s3() {
+    return ['Upload artifacts to s3 bucket': {
+      node(NODE_LINUX_CPU) {
+        ws('workspace/docs') {
+          timeout(time: max_time, unit: 'MINUTES') {
+            if(env.FOLDER_NAME) {
+              utils.unpack_and_init('full_website', 'docs/_build/full_website.tgz')
```

Comment:
can we make sure we have backups and can roll back if there's any issue.

Reply:
Added a back-up mechanism @szha